Aptoide/aptoide-client-v8
# Reactions

**What does this PR do?**

  Final batch of the reaction feature
This PR adds the possibility to do a reaction in the editorial content in home, editorial view and inside a editorial content

**Database changed?**

  No

**Where should the reviewer start?**

- ReactionManager.java
-EditorialPresenter.java
-EditorialBundleViewHolder.java
-EditorialListPresenter.java
-HomePresenter

**How should this be manually tested?**

Do these flows in the 3 screens (Home, Editorial list view, editorial view)

- Do a reaction
- Delete your reaction (Single Press after having a reaction)
- Do another reaction 
- Replace reaction (Long Press)
- Replace reaction with same reaction (should not update view)
- Do 5 reactions (limit, should appear a toast to login when trying to do the 6th)
- Login
- (Reactions you’ve made should disappear and you should be able to react again)

 
**What are the relevant tickets?**

  All tickets done are linked in the following ticket
  https://aptoide.atlassian.net/browse/AS-142


**Code Review Checklist**
